phaseshifts/factories.py

```
@@ -0,0 +1,82 @@
+    pass
+
+from .wrappers import BVHWrapper, EEASiSSSWrapper
+
+
+class PhaseshiftFactory(object):
+    """Class for backend selection"""
+
+    backend = object
+    phsh_files = []  # type: List[str]
+
+    def __init__(self, backend, **kwargs):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)